examples/pcmopus/publishing/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	res "github.com/livekit/mediatransportutil/pkg/audio/res"
+	testdata "github.com/livekit/mediatransportutil/pkg/audio/res/testdata"
+	"github.com/livekit/protocol/logger"
+	lksdk "github.com/livekit/server-sdk-go/v2"
+)
+
+const (
+	host                = "ws://localhost:7880"
+	apiKey              = "devkey"
+	apiSecret           = "secret"
+	roomName            = "test"
+	participantIdentity = "go-sdk-publisher"
+)
+
+func main() {
+	logger.InitFromConfig(&logger.Config{Level: "info"}, "pcmopus")
+	lksdk.SetLogger(logger.GetLogger())
+
+	room, err := lksdk.ConnectToRoom(host, lksdk.ConnectInfo{
+		APIKey:              apiKey,
+		APISecret:           apiSecret,
+		RoomName:            roomName,
+		ParticipantIdentity: participantIdentity,
+	}, nil)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Please avoid `log.Fatal` in any of the code. It just kills the process and doesn't call defers, which causes half-flushed files, missing logs, etc. Using `panic` is preferable.

Reply:
done